.github/workflows/ci.yaml

```diff
@@ -288,52 +292,30 @@ jobs:
         useradd -g adm -s /bin/bash -m chevah
         echo '%adm    ALL=NOPASSWD: ALL' > /etc/sudoers
 
-    # GHA's checkout action fails on CentOS 5/6, issue same commands manually.
-    # This fails when opening a new PR, but works starting with second commit.
-    - name: Clone sources independently
-      run: |
-        cd /home/chevah/
-        git init $CHEVAH_REPO
-        cd $CHEVAH_REPO
-        # Cleanup the repo.
-        git rev-parse --symbolic-full-name --verify --quiet HEAD || true
-        git rev-parse --symbolic-full-name --branches || true
-        git remote remove origin || true
-        # Update repo token.
-        git remote add origin https://github.com/chevah/$CHEVAH_REPO
-        git fetch --no-tags --prune origin
-        # Prepare the code.
-        git clean -f
-        git reset --hard ${{ github.event.after }}
-        git log -1 --format='%H'
-
-    # This action also fails on CentOS 5/6.
+    - uses: actions/checkout@v4
```

Comment:
I went with default git chechout code ... to simplify our GHA yaml

Reply:
The downside is it only supports distributions on which GitHub Actions Runner actually runs.

Reply:
Well... it works on any distribution on which nodejs 16 or 20 can be executed.

I have changed the code and it looks like it also works for Alpine ...for example

GitHub Actions has a separate node_js version compiled for Alpine.

GitHub Actions are doing with nodejs, something similar to what we do with  pythia... they distribute various nodejs versions with each GitHub Action runner.

------

you can mark this as resolved and merge